app/models/manageiq/providers/amazon/manager_mixin.rb

```diff
@@ -68,10 +68,12 @@ def verify_credentials(args)
       region           = args["provider_region"]
       default_endpoint = args.dig("authentications", "default")
 
-      access_key, secret_access_key, proxy_uri, service_account = default_endpoint&.values_at(
-        "userid", "password", "proxy_uri", "service_account"
```

Comment:
Does this mean that proxy_uri doesn't come from DFF _at all_ ?

Reply:
Minor, but if so, we probably should remove line 62 as well.

Reply:
Correct, we don't actually have proxy_uri in the endpoints table so the pre-ddf code must have been updating the settings if you put something here

Reply:
> Minor, but if so, we probably should remove line 62 as well.

:+1: done

Reply:
So I started out by doing `proxy_uri ||= http_proxy_uri` but then noticed there was no proxy_uri in the ddf params, hence removing it from here